lib/framework.js

```diff
@@ -128,6 +128,9 @@ class Framework {
 		if (!this.config.ui5.mode) {
 			this.config.ui5.mode = "html";
 		}
+		if (!this.config.ui5.failOnNoTests) {
```

Comment:
Please add a check so that the new option can only be used with `mode === "html"`.
With script mode it has no effect which might confuse a user.